File: python/bc2060.py
c2 = 0
c3 = 0
c4 = 0
c5 = 0

tVec = int(input())
vec = input().split(" ")

for c in range(0, tVec):
    val = int(vec[c])

    if val % 2 == 0: c2 += 1
    if val % 3 == 0: c3 += 1
    if val % 4 == 0: c4 += 1
    if val % 5 == 0: c5 += 1

print(f"{c2} Multiplo(s) de 2\n{c3} Multiplo(s) de 3\n{c4} Multiplo(s) de 4\n{c5} Multiplo(s) de 5")

File: c99/bc2060.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

int main()
{
    int tVec, c2, c3, c4, c5, i;
    c2 = c3 = c4 = c5 = 0;
    
    scanf("%d", &tVec);
    getchar();
    
    int tMax = tVec * (10+1);
    
    char cVal[tMax];
    int vec[tVec];
    
    fgets(cVal, sizeof(cVal), stdin);
    
    char *token = strtok(cVal, " \n");
    int pos = 0;
    
    while(token != NULL && pos < tVec)
    {
        vec[pos] = atoi(token);
        token = strtok(NULL, " \n");
        pos++;
    }
 
    for(i = 0;i < tVec; i++)
	{
		if(vec[i] % 2 == 0) c2++;
		if(vec[i] % 3 == 0) c3++;
		if(vec[i] % 4 == 0) c4++;
		if(vec[i] % 5 == 0) c5++;
	}
	
	printf("%d Multiplo(s) de 2\n%d Multiplo(s) de 3\n%d Multiplo(s) de 4\n%d Multiplo(s) de 5\n", c2, c3, c4, c5);
    
    return 0;
}


